src/index.js

```diff
@@ -2,7 +2,7 @@ const express = require("express");
 const { ApolloServer, gql } = require('apollo-server-express');
 const { schema } = require("./schema");
 const voyager = require('graphql-voyager/middleware');
-const beehive = require("@wildflowerschools/graphql-beehive")
+const beehive = require("@wildflowerschools/graphql-beehive");
```

Comment:
I just got used to not having the blasted semicolons at the end of lines, and now you want them in there. 😞

Reply:
Lol. I really do not have a preference. Only added for consistency with other lines.

Reply:
I sometimes have to add it to the imports because the interpreter gets confused depending on what the next line is, otherwise I've been actively not putting them in. I used to always put them everywhere, but had worked with some teams that felt strongly about them not being in there.

Reply:
Coming back to this PR after being reminded of it post-tech meeting. Semicolons, ugh. This seems like a good opportunity to decide on an approach so we can capture it in our open source styling guidelines. I feel drawn toward the practice of requiring them so we avoid running into issues where things don't work as one might expect (i.e. to avoid things like the interpreter confusions you allude to). However, I also recognize the larger reason I might side for semis is because I don't have experience NOT including semicolons, which isn't a very good reason. My concern that unintentional issues might crop up is not based on experience but on assumptions. As I've done far less JS programming, I could certainly be persuaded to try styling our JS w/o semicolons.